### PR TITLE
Added Contribution.md

### LINKED ISSUE #20: make a contribution.md for the project
- contribution.md file
- have some basic guidelines, maybe taken from GitHubs examples
- not be to long, people that are interested should be able to read it in a couple of minutes or less
- written in markdown

### PR DESCRIPTION
**Pull request by Petsamuel**
resolved #20

### CODE REVIEW

File: CONTRIBUTION.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Before getting started, please read and adhere to the code of conduct. We want to maintain a respectful and welcoming community for all participants
+
+**Here are the basic steps to contribute to pybot_discord script**
+
+- Fork the repository
+- Create a new branch for your changes
+- Make your changes and commit them to your branch
+- Push your changes to your fork
+- Create a pull request
+- Gave us a star
```
Comment (kmgrime) [inline CONTRIBUTION.md:16]: give*
Nice if you could correct the typo :)

**Review by kmgrime — APPROVED** [review]
LGTM - good job! :) 👍

File: CONTRIBUTION.md (hunk 1 above)
Comment (Petsamuel) [reply CONTRIBUTION.md:16]: sure

**Review by kmgrime — APPROVED** [review]
Great! Thanks.

**Pull request merged**